torch/_inductor/scheduler.py

```diff
@@ -790,8 +878,17 @@ def should_prune(dep: Dep) -> bool:
     "extern_kernels.mm": torch.ops.aten.mm,
     "extern_kernels.bmm": torch.ops.aten.bmm,
     "extern_kernels.addmm": torch.ops.aten.addmm,
+    "aten.mul.Tensor": torch.ops.aten.mul.Tensor,
+    "aten._scaled_dot_product_flash_attention.default": torch.ops.aten._scaled_dot_product_flash_attention.default,
 }
 
+_float_types = [
```

Comment:
can we remove this and use
https://pytorch.org/docs/stable/generated/torch.is_floating_point.html